Translated using Weblate (Russian) for 1.20.1

marisathewitch opened the pull request:
Currently translated at 99.76% (5539 of 5552 strings)

Commit:
Translated using Weblate (Russian)

Currently translated at 99.6% (5531 of 5552 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: http://weblate.pantsu.moe/projects/gregicality/gregtech-ceu-modern-1-20-1/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 99.7% (5538 of 5552 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: http://weblate.pantsu.moe/projects/gregicality/gregtech-ceu-modern-1-20-1/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 99.7% (5539 of 5552 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: http://weblate.pantsu.moe/projects/gregicality/gregtech-ceu-modern-1-20-1/ru/

marisathewitch committed:
Merge branch 'GregTechCEu:1.20.1' into 1.20.1

Commit:
Translated using Weblate (Russian)

Currently translated at 99.7% (5546 of 5559 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: http://weblate.pantsu.moe/projects/gregicality/gregtech-ceu-modern-1-20-1/ru/

marisathewitch committed:
Merge branch 'GregTechCEu:1.20.1' into 1.20.1

screret reviewed (commented):
why are they in a random order?
like, half the ender link lines and 1/2 multiblock screen lines are in the middle of the command description?

Commit:
Translated using Weblate (Russian)

Currently translated at 99.7% (5546 of 5559 strings)

Translation: Gregtech: CEu/GregTech CEu Modern (1.20.1)
Translate-URL: http://weblate.pantsu.moe/projects/gregicality/gregtech-ceu-modern-1-20-1/ru/

Ghostipedia committed:
Merge branch '1.20.1' into 1.20.1
